docs/sources/tutorials/send-logs-to-loki.md

```diff
@@ -179,39 +255,55 @@ The [`loki.process` documentation][loki.process] provides more comprehensive inf
 
 ### Fourth component: Write logs to Loki
 
-Paste this component configuration below the previous component in your `config.alloy` file:
+Copy and paste this component configuration below the previous component in your `config.alloy` file:
 
 ```alloy
-loki.write "grafana_loki" {
-  endpoint {
-    url = "http://localhost:3100/loki/api/v1/push"
-
-    // basic_auth {
-    //  username = "admin"
-    //  password = "admin"
-    // }
+  loki.write "grafana_loki" {
+    endpoint {
+      url = "http://localhost:3100/loki/api/v1/push"
+
+      // basic_auth {
+      //  username = "admin"
+      //  password = "admin"
+      // }
+    }
   }
-}
 ```
 
-This final component creates a [`loki.write`][] component named `grafana_loki` that points to `http://localhost:3100/loki/api/v1/push`.
+This final component creates a [`loki.write`][loki.write] component named `grafana_loki` that points to `http://localhost:3100/loki/api/v1/push`.
 
 This completes the simple configuration pipeline.
 
+<!-- INTERACTIVE ignore START -->
 {{< admonition type="tip" >}}
 The `basic_auth` block is commented out because the local `docker compose` stack doesn't require it.
 It's included in this example to show how you can configure authorization for other environments.
-For further authorization options, refer to the [loki.write][] component reference.
+For further authorization options, refer to the [`loki.write`][loki.write] component reference.
 
 [loki.write]: ../../reference/components/loki/loki.write/
 {{< /admonition >}}
+<!-- INTERACTIVE ignore END -->
+
+{{< docs/ignore >}}
+> The `basic_auth` block is commented out because the local `docker compose` stack doesn't require it. It's included in this example to show how you can configure authorization for other environments.For further authorization options, refer to the [`loki.write`](../../reference/components/loki/loki.write/) component reference.
```

Comment:
```suggestion
> The `basic_auth` block is commented out because the local `docker compose` stack doesn't require it. It's included in this example to show how you can configure authorization for other environments. For further authorization options, refer to the [`loki.write`](../../reference/components/loki/loki.write/) component reference.
```